packages/payload-cloud/src/plugin.ts

```diff
@@ -93,5 +98,72 @@ export const payloadCloudPlugin =
       })
     }
 
+    // We make sure to only run cronjobs on one instance using a instance identifier stored in a global.
+
+    const DEFAULT_CRON = '* * * * *'
+    const DEFAULT_LIMIT = 10
+    const DEFAULT_CRON_JOB = {
+      cron: DEFAULT_CRON,
+      limit: DEFAULT_LIMIT,
+      queue: 'default (every minute)',
```

Comment:
Now that cronjobs are defined in config.jobs.autoRun and not in the plugin, where this "default cron job" should be documented. I copy the internal question:

> Should the default cronjob:
> 1. be set always after jobs.autoRun if it returns an empty array []?
> 2. be set only in Payload Cloud Plugin if jobs autoRun returns an empty array?
> 3. there should not be a default cronjob.
> 
> Personally I like option 3 better.
> If we go with option 2 where would it be documented in tsdoc? the payload cloud plugin will not have any option to configure cron jobs, just to enable or disable it